net.sf.VICE.yml

```diff
@@ -13,7 +13,6 @@ build-options:
 finish-args:
   - --device=all
   - --filesystem=home:ro
-  - --filesystem=~/.vice:create
```

Comment:
@Compyx Migration logic for this would be appreciated also.

If `XDG_CONFIG_HOME` doesn't exist and `~/.vice` or `~/.config/vice` does, copy over.